docs/streams/upgrade-guide.html

```diff
@@ -38,6 +44,16 @@
         See <a href="#streams_api_changes_0101">below</a> a complete list of 0.10.1 API changes that allow you to advance your application and/or simplify your code base, including the usage of new features.
     </p>
 
+    <h3><a id="streams_api_changes_100" href="#streams_api_changes_100">Streams API changes in 1.0.0</a></h3>
+    <!-- TODO: KIP-120, 138, 160, 161, 167, 173 -->
```

Comment:
I agree. In fact the goal moving forward is that the code changes and the doc changes should be going in single PR or consecutive PRs if a single one is too large. And we should not mark the KIP / JIRA as resolved until it is done so that we would not need to such "house cleaning" later one towards the release deadline.

These KIPs are already merged in some time ago until I realized it was missing, and we are adding them just as a one-time thing.